apps/prairielearn/elements/pl-dropdown/pl-dropdown.py

```diff
@@ -58,7 +58,7 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
     pl.check_attribs(
         element,
         required_attribs=["answers-name"],
-        optional_attribs=["blank", "allow-blank", "weight", "sort"],
+        optional_attribs=["blank", "allow-blank", "weight", "sort", "aria-labelledby"],
```

Comment:
You didn't test these changes. I see the following when I try to open `exampleCourse/questions/element/dropdown`:

<img width="983" alt="Screenshot 2025-04-28 at 08 10 05" src="https://github.com/user-attachments/assets/67e8b847-ff1e-40ed-bb8e-143463a3d087" />

Reply:
Sorry about that, I forgot that when I pulled them out of #11837 they don't get tested anymore.

Reply:
[3f527d0](https://github.com/PrairieLearn/PrairieLearn/pull/11844/commits/3f527d0bd67b8c09d9e8416a41c6855d08e6f262)

Reply:
![image](https://github.com/user-attachments/assets/fb84d7e8-452a-46e3-8a8a-7b309f783c7f)